es-models/gdc_from_graph/file.mapping.yaml

```diff
@@ -0,0 +1,1290 @@
+          tissue_or_organ_of_origin:
+            type: keyword
+          treatments:
+            type: nested
+            properties:
+              created_datetime:
+                type: keyword
+              days_to_treatment:
+                type: long
+              days_to_treatment_end:
+                type: long
+              days_to_treatment_start:
+                type: long
+              state:
+                type: keyword
+              submitter_id:
+                type: keyword
+              therapeutic_agents:
+                type: keyword
+              treatment_anatomic_site:
+                type: keyword
+              treatment_id:
+                type: keyword
+              treatment_intent_type:
+                type: keyword
+              treatment_or_therapy:
+                type: keyword
+              treatment_outcome:
+                type: keyword
+              treatment_type:
+                type: keyword
+              updated_datetime:
+                type: keyword
+          tumor_grade:
+            type: keyword
+          tumor_stage:
+            type: keyword
+          updated_datetime:
+            type: keyword
+          vascular_invasion_present:
+            type: keyword
+          vital_status:
+            type: keyword
+          year_of_diagnosis:
+            type: long
+      disease_type:
+        type: keyword
+      exposures:
+        type: nested
+        properties:
+          alcohol_history:
+            type: keyword
+          alcohol_intensity:
+            type: keyword
+          bmi:
+            type: long
+          cigarettes_per_day:
+            type: long
+          created_datetime:
+            type: keyword
+          exposure_id:
+            type: keyword
+          height:
+            type: long
+          pack_years_smoked:
+            type: long
+          state:
+            type: keyword
+          submitter_id:
+            type: keyword
+          tobacco_smoking_onset_year:
+            type: long
+          tobacco_smoking_quit_year:
+            type: long
+          tobacco_smoking_status:
+            type: keyword
+          updated_datetime:
+            type: keyword
+          weight:
+            type: long
+          years_smoked:
+            type: long
+      family_histories:
+        type: nested
+        properties:
+          created_datetime:
+            type: keyword
+          family_history_id:
+            type: keyword
+          relationship_age_at_diagnosis:
+            type: long
+          relationship_gender:
+            type: keyword
+          relationship_primary_diagnosis:
+            type: keyword
+          relationship_type:
+            type: keyword
+          relative_with_cancer_history:
+            type: keyword
+          state:
+            type: keyword
+          submitter_id:
+            type: keyword
+          updated_datetime:
+            type: keyword
+      files:
```

Comment:
Similar to what's commented above in `case` index, this `file.cases.files` node is not need. Should be removed.